### DISCUSSION TITLE
Is there a way to disable icon "#" for headers?

### DISCUSSION THREAD

**Discussion by podloznyi** [retypeapp/retype#476] [Q&A] [answered]
Is there a way to disable icon "#" for headers, on a publicly available pages or overall?
![image](https://user-images.githubusercontent.com/60586595/212749662-4e46104c-a24a-454d-b770-1a383ac589d3.png)

**Comment by geoffreymcgill** [comment, marked as answer]
No, sorry. There are no built in features within Retype to disable those anchors.
